tests/stdlib/test_math.py

```diff
@@ -0,0 +1,21 @@
+from ..utils import TranspileTestCase
+
+
+class MathModuleTests(TranspileTestCase):
```

Comment:
Testcases for integers and and at least one disallowed type eg strings would be nice.

Reply:
Our initiall idea was that `math.fabs()` is a function that takes a float and returns a float, by definition of the function. However looking close on it it seems that it takes a `number` and returns a `float`, so testing with an integer might be good.
Is it necessary to test fo disallowed values, like a `string`,  though? Thinking by definition of the input  `math.fabs()`

Reply:
What Raphaela said when asked about how extensive the tests should be: 

"Your implementations and tests should cover the functionality with its inputs and outputs. So you should e.g. test a number of different inputs and types. Covering exceptional behaviour would be good too, if applicable. 

Since you don't have unlimited time for this, make sure that you cover the frequent use cases and try to show some diversity - e.g. if you take math.cos() it is enough if you give one example of the function's behaviour for non-numerical types (e.g. string), you don't have to test *every* type in the Python standard library. "

Reply:
Extra test cases for integers and one for an unsupported type were added in 365413d. The functionality for type checking was added in adbf7d2.